src/patterns/ec2-app.ts

```diff
@@ -170,16 +173,13 @@ export interface GuEc2AppProps extends AppIdentity {
   userData: GuUserDataProps | string;
   access: AppAccess;
   applicationPort: number;
-  certificateProps: GuDomainNameProps;
```

Comment:
Renaming `certificateProps` to `domain` is the only breaking change to the API of this pattern. If we didn't rename it, those that are only using the pattern have a no effort upgrade path. Shall we do that?

Reply:
I agree that the new name is better, but perhaps we should make this breaking change when we remove stage-awareness from the patterns to avoid changing the API twice in quick succession?

Reply:
Reverted.